examples/golang/append_object.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/smithy-go/ptr"
+
+	"log"
+	"os"
+)
+
+func uploadObject(bucketName, objectName string) {
+	// Open an object:
+	obj, err := os.Open(objectName)
+	if err != nil {
+		log.Println("Open file failed:", err)
+	}
+	defer obj.Close()
+
+	// Create a S3 session:
+	s3session := s3.New(session.Must(session.NewSession(&aws.Config{
+		Region:           aws.String("default"),
+		Endpoint:         aws.String("http://127.0.0.1:8000"),
```

Comment:
this endpoint is specific to vstart. you should pass that to this function (you can hardcode that in the main function)

Reply:
Thanks very much for the comments! I will hardcode the endpoint parameter in the main function. 

I think it is better to hardcode all vstart parameters in the main function and pass s3session as a parameter for uploadObject() and listObject() functions?

Reply:
sounds good